meson.build

```diff
@@ -3,6 +3,7 @@ project('ubsh', 'c',
     'warning_level=3',
     'debug=true',
     'b_sanitize=address',
+    'std=gnu89',
```

Comment:
Do we really want C89 haha

Reply:
Yes.

**gnu89** (not C89) gives you the same features as C99 and it enforces that you declare variables at the start of functions, making for cleaner code.